crypto/elliptic-curve.unit-test.ts

```diff
@@ -164,3 +165,78 @@ function curveWithFields(params: CurveParams) {
   // return Curve, Field and Scalar
   return [Projective, Affine.Field, Affine.Scalar] as const;
 }
+
+// Twisted Edwards curve tests
```

Comment:
Can you have some test vectors? I.e. some points defined in the specification that we know are on the curves, in the prime subgroups, and some known results and inputs for each operation?